src/components/BottomBar/SkyBrowser/SkyBrowserTabs.jsx

```diff
@@ -252,7 +341,7 @@ class SkyBrowserTabs extends Component {
                   if (!e) var e = window.event;
```

Comment:
Here is the same code structure that I saw before and found weird
```
if (!e) var e = window.event;
e.cancelBubble = true;
if (e.stopPropagation) e.stopPropagation();
```

What does it do, and does it have to be written like this?